googleapiclient/discovery_cache/appengine_memcache.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2014 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""App Engine memcache based cache for the discovery document."""
+
+
+import logging
+
+# This is only an optional dependency because we only import this
+# module when google.appengine.api.memcache is available.
+from google.appengine.api import memcache
+
+from . import base
+
+NAMESPACE = 'google-api-client'
+
+
+class Cache(base.CacheBase):
+  """A cache with app engine memcache API.
+
+  Attributes:
+    max_age: Cache expiration in seconds.
+  """
+  def __init__(self, max_age):
+      """Constructor for appengine_memcache.Cache.
+
+      Args:
+        max_age: Cache expiration in seconds.
+      """
+      self.max_age = max_age
+
+  def get(self, url):
+    """Gets the content from the memcache with a given key.
```

Comment:
No need for a doc string here unless this method behaves in a way that couldn't be understood from the doc string of the abstract method that it implements.